zap/src/main/java/org/parosproxy/paros/Constant.java

```diff
@@ -185,6 +185,15 @@
 
     private static final String VERSION_ELEMENT = "version";
 
+    public static enum OS {
+        Windows,
+        Linux,
+        MacOS,
+        Kali,
+        BackBox,
```

Comment:
I've never heard of this?

Reply:
https://www.backbox.org/ and they've updated to use the latest ZAP :)

Reply:
Huh, okay cool.